app/ui/src/app/integration/edit-page/save-or-add-step/save-or-add-step.component.html

```diff
@@ -1,90 +1,90 @@
-                  <a [routerLink]="['/']">Home</a>
-                </li>
-                <li>
-                  <a [routerLink]="['/integrations']">Integrations</a>
-                </li>
-                <li>
-                  <a *ngIf="flowPageService.integrationName"
-                     [routerLink]="['/integrations', currentFlowService.integration?.id]">{{ flowPageService.integrationName }}</a>
-                  <ng-container *ngIf="!flowPageService.integrationName">New Integration</ng-container>
-                </li>
-                <li class="active">Save or Add Step</li>
-              </ol>
-            </div>
-            <div class="toolbar-pf-action-right">
-              <button class="btn btn-default"
-                      (click)="cancel()">Cancel
-            </button> &nbsp;
-              <button class="btn btn-default"
-                      (click)="save()"
-                      [disabled]="!flowPageService.canContinue() || flowPageService.saveInProgress || flowPageService.publishInProgress"><span *ngIf="flowPageService.saveInProgress" class="spinner spinner-sm spinner-inline"></span>Save as Draft
-            </button>
-              <button type="button"
-                      class="btn btn-primary"
-                      (click)="publish()"
-                      [disabled]="!flowPageService.canContinue() || flowPageService.saveInProgress || flowPageService.publishInProgress"><span *ngIf="flowPageService.publishInProgress" class="spinner spinner-sm spinner-inline"></span>Publish</button>
+      <div class="container-fluid">
+        <div class="row toolbar-pf">
+          <div class="col-sm-12">
+            <div class="toolbar-pf-actions">
+              <div class="inline-block">
+                <ol class="breadcrumb">
+                  <li>
+                    <a [routerLink]="['/']">Home</a>
+                  </li>
+                  <li>
+                    <a [routerLink]="['/integrations']">Integrations</a>
+                  </li>
+                  <li>
+                    <a *ngIf="flowPageService.integrationName"
+                      [routerLink]="['/integrations', currentFlowService.integration?.id]">{{ flowPageService.integrationName }}</a>
+                    <ng-container *ngIf="!flowPageService.integrationName">New Integration</ng-container>
+                  </li>
+                  <li class="active">Save or Add Step</li>
+                </ol>
+              </div>
+              <div class="toolbar-pf-action-right">
+                <button class="btn btn-default"
+                        (click)="cancel()">Cancel
+              </button> &nbsp;
+                <button class="btn btn-default"
+                        (click)="save()"
+                        [disabled]="!flowPageService.canContinue() || flowPageService.saveInProgress || flowPageService.publishInProgress"><span *ngIf="flowPageService.saveInProgress" class="spinner spinner-sm spinner-inline"></span>Save as Draft
```

Comment:
I'm surprised this LOC passed the `yarn lint` gate, since it clearly exceeds the linter max-line settings. Coudl you properly reformat this for better readability? Thanks.